src/pages/Admin/Venue/VenueDetailsSubForm.tsx

```diff
@@ -97,9 +98,13 @@ export const VenueDetailsSubForm: React.FC<VenueDetailsSubFormProps> = ({
   formError,
   setFormError,
 }) => {
+  const { worldSlug } = useWorldParams();
   const values = watch();
   const urlSafeName = values.name
```

Comment:
```suggestion
  const urlSafeName = generateAttendeeLandingUrl({
        worldSlug,
        spaceSlug: createSlug<SpaceSlug>(values.name),
        default: undefined, // in case of missing slug, might also be "#" or "/"
        absolute: true,
      })
```